tests/dispense.rs

```diff
@@ -253,10 +277,76 @@ async fn many_concurrent_requests() {
         .into_iter()
         .filter(|tx| !matches!(tx.transaction, TransactionType::Mint(_)))
         .collect::<Vec<_>>();
+    assert_eq!(3, txs.len());
 
-    assert_eq!(COUNT, txs.len());
-    assert_eq!(
-        test_balance,
-        COUNT as u64 * context.faucet_config.dispense_amount
-    );
+    for recipient in recipient_addresses {
+        let test_balance = context
+            .provider
+            .get_asset_balance(&recipient.into(), context.faucet_config.dispense_asset_id)
+            .await
+            .unwrap();
+
+        assert_eq!(test_balance, context.faucet_config.dispense_amount);
+    }
+}
+
+#[tokio::test]
+async fn dispense_once_per_day() {
+    let mut rng = StdRng::seed_from_u64(42);
+    let recipient_address: Address = rng.gen();
+    let recipient_address_str = format!("{}", &recipient_address);
+    let context = TestContext::new(rng).await;
+    let addr = context.addr;
+
+    let dispense_interval = 24 * 60 * 60;
+    let time_increment = dispense_interval / 6;
+
+    let response = reqwest::Client::new()
+        .post(format!("http://{addr}/dispense"))
+        .json(&json!({
+            "captcha": "",
+            "address": recipient_address_str.clone(),
+        }))
+        .send()
+        .await
+        .expect("First dispensing request should be successful");
+
+    assert_eq!(response.status(), reqwest::StatusCode::CREATED);
+
+    for _ in 0..5 {
+        //tokio::time::advance(Duration::from_secs(time_increment)).await;
```

Comment:
It looks like you can remove this line (and the other commented out lines below)